src/main/java/com/epam/aidial/core/service/ResourceService.java

```diff
@@ -522,6 +713,60 @@ private static long time() {
         return System.currentTimeMillis();
     }
 
-    private record Result(String body, long createdAt, long updatedAt, boolean synced, boolean exists) {
+    private void flushToBlobStore(ResourceDescription descriptor) {
+        RMap<String, byte[]> map = sync(redisKey(descriptor));
+        map.delete();
+    }
+
+    private String getEtag(ResourceDescription descriptor) {
```

Comment:
Just use metadata and remove  this method

Reply:
I use it in 2 places now.